.github/workflows/vercel-preview-pr-target.yaml

```
@@ -0,0 +1,90 @@
+        working-directory: packages/webapp
+        env:
+          PUBLIC_WALLETCONNECT_PROJECT_ID: test
+
+      - name: Install Vercel CLI (local, pinned)
+        shell: bash --noprofile --norc -euo pipefail {0}
+        run: |
+          VERCEL_DIR="$(mktemp -d)"
+          npm install --no-audit --no-fund --no-save --ignore-scripts --prefix "$VERCEL_DIR" vercel@33.4.1
+          echo "VERCEL_BIN=$VERCEL_DIR/node_modules/.bin/vercel" >> "$GITHUB_ENV"
+          test -x "$VERCEL_DIR/node_modules/.bin/vercel"
+        shell: bash --noprofile --norc -euo pipefail {0}
```

actionlint (1.7.7)

key "shell" is duplicated in element of "steps" section. previously defined at line:70,col:9

(syntax-check)